test/MUXControlsTestApp/MainPage.xaml

```diff
@@ -61,7 +61,7 @@
         </ScrollViewer>
         <StackPanel Orientation="Horizontal" Grid.Row="2" HorizontalAlignment="Center">
             <CheckBox x:Name="LongAnimationsDisabled" AutomationProperties.AutomationId="LongAnimationsDisabled" Content="Long Animations Disabled" Checked="LongAnimationsDisabled_Checked" Unchecked="LongAnimationsDisabled_Unchecked" Margin="10,0,0,0"/>
-
+            <Button x:Name="GarbageCollection" Content="GarbageCollection" VerticalAlignment="Center" Click="GarbageCollection_Click" Margin="5"/>
```

Comment:
Should this be available when within a test page as well? like the render in lab frame dimensions button?

Reply:
I added this here so you can hit the button, go to a test page, come back, hit the button again, and there should be no leak.

Reply:
OK - I moved it to the TestFrame level.